test/extended/cluster/cl.go

```diff
@@ -0,0 +1,147 @@
+package cluster
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	oapi "github.com/openshift/origin/pkg/api"
+	projectapi "github.com/openshift/origin/pkg/project/api"
+	exutil "github.com/openshift/origin/test/extended/util"
+	testutil "github.com/openshift/origin/test/util"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+)
+
+const deploymentRunTimeout = 5 * time.Minute
+
+var _ = g.Describe("clusterloader", func() {
```

Comment:
depends on if you're using our image-provisioner/aos-ansible (or somehow else pre-pulling all content into your docker graph for every node).  If so, just a couple of minutes the default config I suggested that @sjug send in this PR is very basic.

When I was describing it to you the other day I mentioned that you can modulate the complexity/run-time simply by adjusting the number of projects within the config file.